# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../heimdal/no-applications.patch
# Copyright (C) 2010 - 2014 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description: Do not build/install all the "kerberized" applications that
	     are shipped with heimdal

diff --git a/Makefile.am b/Makefile.am
index 2508096..912c15a 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -7,7 +7,7 @@ kcm_dir = kcm
 endif
 
 SUBDIRS=  include base lib kuser kdc admin kadmin kpasswd 
-SUBDIRS+= $(kcm_dir) appl tools tests packages etc po
+SUBDIRS+= $(kcm_dir) tools tests packages etc po
 
 if HEIMDAL_DOCUMENTATION
 SUBDIRS+= doc
